tests/transforms/test_experimental/test_transform_dispatcher.py

```diff
@@ -265,6 +265,30 @@ def qnode_circuit(a):
         assert transform_container.classical_cotransform is None
         assert not expand_transform_container.is_informative
 
+    def test_transform_program_with_expand_transform(self):
+        """Test transform with expand transform on a transform program."""
```

Comment:
this test doesn't really clarify what's going on to me. can you check more of what each popped element is?

Reply:
You apply the transform on an empty program, it adds the expand transform and the transform itself. The test simply checks it is done correctly.